versioning-policy.md

```diff
@@ -0,0 +1,32 @@
+## Version Numbering
+
+We will follow [SemVer](https://semver.org/)’s Major.Minor.Patch format, defined as follows: 
+- **Major**: increments upon _breaking changes_ - ie, changes that would cause errors in implementations relying on the current latest version
+    - Removing or renaming a field
+    - Changing a field’s type, or removing previously-available enum values
+    - Adding new field(s)/values which cannot be mapped to previous values
+- **Minor**: increments upon the addition of new functionality that maintains _forward compatibility_ - ie, existing implementations can continue without error
+    - Adding a new field, adding values to an existing enum
+- **Patch**: a change which does not meaningfully alter API output - for example, bugfixes that alter incorrect behaviour to align with documented behaviour
+
+## Version Lifecycle
+
+There are several stages of a version lifecycle, specified by `version_status` field in WCIF. A nullable `status_advancement_date` field will indicate when the `version_status` will progress to the next stage of the lifecycle.
+- **Beta**: Early release of a new version - made available to developers for testing, or early access to newly-supported features. Spec may be subject to change
+- **Latest**: Latest WCIF version, made available to developers for feedback and early migration. Spec is considered final.
+    - When a `latest` version is released, a `status_advancement_date` will be given to the current `stable` version
+- **Stable**: The WCIF version served by default
+- **Deprecated**: Backwards compatibility will be maintained where possible, but clients should updated their version to either `latest` (ideally) or `stable` (at minimum) by the `status_advancement_date` to maintain functionality
+- **Removed**: No longer supported, not possible to access via API endpoints
+
+## Requesting Specific Versions
+The existing WCIF endpoints will remain, and always serve the `stable` version of WCIF.
+
+Two new ways of querying WCIF will be added: 
+- `api/v0/competitions/{competition-id}/wcif/{lifecycle-name}` to request the latest, or beta versions
+- `api/v0/competitions/{competition-id}/wcif/version/{version-number}` to request a specific version number
```

Comment:
Do we have a corresponding PR on the main website to actually provide these APIs?